### ISSUE TITLE
Eye of Ender not properly locating Stone Brick Stronghold

### ISSUE THREAD

**Issue by KimJongUninstall** [TelepathicGrunt/RepurposedStructures#60]
Just as the title says, created a new world, and can't use eye of ender to locate the stronghold. the /locate command works fine though.

**Comment by TelepathicGrunt** [comment]
what versions of repurposed structures are you using?

**Comment by TelepathicGrunt** [comment]
Also, a list of mods you have one will greatly help as well. It could be another mod messing with the eyes of ender in a way that makes my mixin no longer work

**Comment by TelepathicGrunt** [comment]
it's been 11 days and no response lol. Gonna close this now as I cannot fix what I cannot reproduce. I need info in order to figure out the issue

**Comment by Yupoman** [comment]
The same thing happens to me, it points to the place where a stronghole originally appears in the same seed in vanilla (I checked it in another world) but in the world with mods there is nothing there and the eye keeps pointing out that the stronghole is there.
repurposed_structures_forge-5.1.14+1.18.2
Forge 40.1.19

**Comment by TelepathicGrunt** [comment]
@Yupoman are you positive this is related to RS? I do not touch the vanilla stronghold at all. If you have another mod that touches the eyes of ender, they are probably going to be more likely the cause.

testing with just 1.18.2 rs alone and the eyes are working just fine so I doubt it’s an issue with my mod at this point unless you can narrow it down to certain mods in your modpack

**Comment by TelepathicGrunt** [comment]
Also, if you have better strongholds mod on, they stop vanilla strongholds from spawning so the eyes should be pointing to their new stronghold locations. A mod may be removing the vanilla stronghold but the eye isn’t changed properly to point to the new structure. Maybe end remastered might be involved. Either way, doubtful it’s anything to do with rs

**Comment by Yupoman** [comment]
The truth is that I have no idea, I have many mods but for structures only this one, Quark (add dungeons) and for biomes I have Biomes O' Plenty and Oh The Biomes You'll Go. I just comment here since it is the only one where I found someone with the same problem and this mod adds strongholds to the end and nether, but if it doesn't touch those of the overworld then it shouldn't be.
Surely it was the new biomes that overwrote the structure. When I created the world I didn't have the mods installed yet, although the stronghold is far away I guess the game decides where they will appear from the beginning, even if the terrain hasn't even been created, and that broke eyes of ender.
Thanks for responding and sorry for the inconvenience.
